pyproject.toml

```diff
@@ -12,23 +12,23 @@ keywords = ["code review", "gitlab"]
 reviewcheck = "reviewcheck.app:run"
 
 [tool.poetry.dependencies]
-python = "^3.6.9"
-requests = "^2.27.1"
-pyyaml = "^5.1"
-shtab = "^1.5.4"
-colorama = "^0.4.4"
-rich = "^12.4.4"
+python = ">=3.8.1"
```

Comment:
Kind of think it would be nice to mention in the README.md what Python versions we support. Either in the installation guide or just a small entry in the question list. Do you agree?

Reply:
I agree. Added a bit to the README.

Reply:
Great!